workflow/exp/exp.ens_conus12km

```diff
@@ -1,26 +1,66 @@
 #!/user/bin/env bash
 # Aloha! This file is to setup some top level experiment options
-#
-export OPSROOT=/lfs5/BMC/wrfruc/gge/nco
-export EXP_NAME=test1
+
 export NET=rrfs                    # rrfs, rtma, 
 export MESH_NAME=conus12km         # conus12km, conus3km, atl3km, etc
-export VERSION=1.0.1
-export TAG=c1v01
-#
-export EXPDIR=${OPSROOT}/exp/${VERSION}/${EXP_NAME}
-export COMROOT=${OPSROOT}/${EXP_NAME}/com      # task input and output data as well as logs
-export DATAROOT=${OPSROOT}/${EXP_NAME}/stmp    # task workdirs ($DATA) which to be removed immediately upon task completion unless KEEPDATA=yes
+export WGF="ensf"   # working group function = "det", "enkf", "ensf", "firewx"
```

Comment:
@BenjaminBlake-NOAA @MatthewPyle-NOAA @hu5970 
I really think it is a good time to change `ensf` to `enfcst` as suggested by @BenjaminBlake-NOAA in https://github.com/NOAA-EMC/rrfs-workflow/pull/550#discussion_r1848239736.

rrfsv2 and rrfsv1 are different systems, we may not need to follow the misleading use of `ensf`

Reply:
We are now working on "enkf" even we will not add EnKF analysis step for now.

Reply:
But line 6 defines `ensf` instead of `enkf`. Should it be `enkf`?

Reply:
Changed it to enkf. Thanks!

Reply:
Haha, this demonstrated one can easily get confused between `enkf` and `ensf`

Reply:
If we are not limited to 4 characters, ensda and enfcst (or ensfcst) makes more sense to me.